src/client/src/components/pages/loading/index.css

```diff
@@ -0,0 +1,111 @@
+.loading {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+}
+
+.loading-img {
+  width: 250px;
+  height: 250px;
+}
+.loader {
+  color: #e06c00;
+  font-size: 90px;
+  text-indent: -9999em;
+  overflow: hidden;
+  width: 1em;
+  height: 1em;
+  border-radius: 50%;
+  margin: 72px auto;
+  position: relative;
+  -webkit-transform: translateZ(0);
+  -ms-transform: translateZ(0);
+  transform: translateZ(0);
+  -webkit-animation: load6 1.7s infinite ease, round 1.7s infinite ease;
+  animation: load6 1.7s infinite ease, round 1.7s infinite ease;
+}
+@-webkit-keyframes load6 {
```

Comment:
I think create-react-app includes [autoprefixer](https://github.com/postcss/autoprefixer) by default, which means you wouldn't have to add the vendor prefixes like this anywhere.
You could probably do a quick test to verify - delete a `-webkit` style, run the app, and inspect the element chrome and see if your build process has added the `-webkit` rule back to the element.